server/views/components/need-score/template.njk

```diff
@@ -4,7 +4,7 @@
 
 {% set barTypeClass = 'needs-score-bar--small-' + needsScore.score + ' needs-score-bar--small-tenths' %}
 
-<div class="needs-score govuk-body">
+<div class="needs-score govuk-body" aria-hidden='true'>
```

Comment:
double-quotes for consistency with the rest of the HTML markup please.

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/442/commits/62fa9e71a7be4b8c2e64fc8707ab8debfd2485ab